### DISCUSSION TITLE
Development of a GUI

### DISCUSSION THREAD

**Discussion by Sporknife** [liquidctl/liquidctl#296] [Ideas]
Yes, I know that there is an already existing issue but its old (2018). So here it goes.                                                                                 
I'm working on a GUI for liquidctl made in with Qt. Feel free to join me in this [this](https://discord.gg/xW5EYb) discord server.  
Currently its in early development and i am working on creating all widgets/elements in Qt Designer or in Python (whatever is better).

**Comment by cluxter** [comment]
This sounds like a nice project. Do you have a Git repo somewhere to which we could contribute?

**Comment by Sporknife** [comment]
@cluxter yea i do. I made it public again some time ago. Repo linked [here](https://github.com/Sporknife/Liquidctl-Qt). Sadly its not as nearly as completed as i want it to.

**Comment by jonasmalacofilho** [comment]
Issue moved to discussions during triage.

**Comment by jonasmalacofilho** [comment]
Another GUI project in the works, by @sgehrman:

https://github.com/sgehrman/liquid_ui

**Comment by bandaid** [comment]
UI for Windows would be nice?

**Reply by sgehrman** [reply]
My GUI is written in Flutter.  It should work on Windows?   I'm calling the liquidctl tool from my code and you have to have that installed to work.  I'm not sure how Windows handles that.  I assume it will work, but didn't test it.

get the code and try it on windows.  Fork it or send me a PR and I can merge any changes you need for Windows.

https://github.com/sgehrman/liquid_ui

**Comment by NeleBiH** [comment]
If someone is still interested for this i done it but with AI so you know i am not coder  
https://github.com/NeleBiH/LiquidctlGUI/tree/main feel free to improve it if you can
<img width="804" height="526" alt="1" src="https://github.com/user-attachments/assets/2d389dcb-6ee7-45af-9d7f-2b38058df2b1" />
<img width="799" height="516" alt="Screenshot_20250717_232045" src="https://github.com/user-attachments/assets/2036d3dc-527f-4b38-9c62-c42f2c60321e" />
